biostructbenchmark/cli.py

```diff
@@ -0,0 +1,60 @@
+"""CLI scripts called from __main__.py"""
+
+import argparse
+import os
+from Bio.PDB import *
+import ast
+
+
+def validate_file_path(file_path: str) -> str:
+    "Validate file_path input"
+    if os.path.isfile(file_path):
+        return file_path
+    else:
+        raise argparse.ArgumentTypeError(f"{file_path} is not a valid file path")
+
+
+def file_type(file_type: str) -> str:
+    """Ensure input file_type is mmcif or pdb"""
+    if file_type.lower() == "mmcif":
+        return "MMCIF"
+    if file_type.lower() == "pdb":
+        return "PDB"
+    else:
+        raise argparse.ArgumentTypeError(
+            f"file_type argument must be either MMCIF or PDB"
+        )
+
+
+def get_version() -> str:
+    """Get version from __init__.py"""
+    for line in open("biostructbenchmark/__init__.py"):
+        if line.startswith("__version__ = "):
+            return ast.literal_eval(line.split("=")[1].strip())
+    return "Undefined version"
+
+
+def arg_parser():
```

Comment:
Typing is likely `argparse.Namespace`, can check with `print(type())`. There is some special way to call types from imports. Skipping this one is okay, lint later :)